asset/index.html

```diff
@@ -0,0 +1,404 @@
+                + '<div class="md-2">'
+                + '<label class="mt-1">Download a meshagent configuration from</label>'
+                + '<div class="form-outline">'
+                + '<div class="text-box">'
+                + '<div class="text-container">'
+                + '<span class="text-icon">'
+                + '<i class="far fa-copy" onclick="copy(\'#session-config\')"></i>'
+                + '</span>'
+                + '<textarea class="form-control" id="session-config" rows="1" readonly>'
+                + agentMSH
+                + '</textarea>'
+                + '</div>'
+                + '</div>'
+                + '</div>'
+                + '</div>'
+                + '<div>'
+                + '<label>Start the session on Windows from the PowerShell</label>'
+                + '<div>'
+                + '<div>'
+                + '<div class="text-box">'
+                + '<div class="text-container">'
+                + '<span class="text-icon">'
+                + '<i class="far fa-copy" onclick="copy(\'#windows-session\')"></i>'
+                + '</span>'
+                + '<textarea class="form-control" id="windows-session" rows="3" readonly>'
+                + 'wr -Uri https://github.com/Ylianst/MeshCentral/raw/master/agents/MeshService-signed.exe -OutFile meshagent.exe\n'
+                + 'iwr -Uri'
+                + agentMSH
+                + ' -OutFile meshagent.msh\n'
+                + '.\\meshagent.exe connect"'
+                + '</textarea>'
+                + '</div>'
+                + '</div>'
+                + '</div>'
+                + '</div>'
+                + '</div>'
+                + '<div>'
+                + '<label class="label-on-border">Start the session on Linux from the Terminal</label>'
+                + '<div class="text-box">'
+                + '<div class="text-container">'
+                + '<span class="text-icon">'
+                + '<i class="far fa-copy" onclick="copy(\'#linux-session\')"></i>'
+                + '</span>'
+                + '<textarea class="form-control" id="linux-session" rows="4" readonly>'
+                + 'curl https://raw.githubusercontent.com/Ylianst/MeshCentral/master/agents/meshagent_x86-64 -O meshagent\n'
+                + 'chmod +x meshagent.msh\n'
+                + 'curl '
+                + agentMSH
+                + ' -OutFile meshagent.msh\n'
+                + '.\\meshagent connect"'
+                + '</textarea>'
+                + '</div>'
+                + '</div>'
+                + '</div>'
+                + '</div>'
+            $('#session-details').html(dHTML);
+        }
+
+        function deleteSession(id) {
+            console.log(id)
+            if (confirm('Are you sure you want to delete session ' + id + '?')) {
+                $.ajax({
+                    type: 'DELETE',
+                    url: 'https://localhost:8080/session/' + id,
```

Comment:
Plexus isn't required to run on localhost:8080, see how the fetch is done in the share.html.